mit-bitcoin-expo/mit-bitcoin-expo-2019/signature-scheme-security-properties.md

```diff
@@ -1,8 +1,10 @@
 ---
-title: Signature Scheme Security Properties
+title: Andrew Poelstra - Secure Signatures (2019-03-09)
 transcript_by: Bryan Bishop
 categories: ['conference']
-tags: ['security', 'schnorr']
+tags: ['schnorr', 'multisig']
+speakers: ['Andrew Poelstra']
+date: 2019-03-09
```

Comment:
this needs to be removed. see #188

Reply:
Pushed an additional commit removing the date from the header

Reply:
Now removed that commit